### ISSUE TITLE
Issues with Random

### ISSUE THREAD

**Issue by meowingtwurtle** [Nybbit/NybUtils#1]
Random should not have an user-declared destructor, as it has an empty body, and doing do prevents automatic generation of the move constructor and assignment operators, as well as prevents certain optimizations by preventing it from being trivially destructbile.

Does it make sense for Random to by copyable and/or movable?

**Comment by Nybbit** [comment]
Yep, you're right, there shouldn't be a user-declared destructor. I'll probably remove this repo because the functionality it adds is so trivial.
